lib/credit_card_sanitizer.rb

```diff
@@ -44,13 +44,16 @@ class CreditCardSanitizer
   NONEMPTY_LINE_NOISE = /#{LINE_NOISE_CHAR}{1,5}/
   SCHEME_OR_PLUS = /((?:&#43;|\+)|(?:[a-zA-Z][\-+.a-zA-Z\d]{,9}):[^\s>]+)/
   NUMBERS_WITH_LINE_NOISE = /#{SCHEME_OR_PLUS}?\d(?:#{LINE_NOISE}\d){10,30}/
+  HTML_TAGS = /\w+((\s+\w+(\s*=\s*(?:"(.|\n)*?"|'(.|\n)*?'|[^'">\s]+))?)+\s*|\s*)/
```

Comment:
This needs to start with a `<` character and end with a `>` character

Right now, it's matching things it should not. For instance, the string `hello` matches this regex. (Try it in rubular)

This isn't good because matching more stuff than expected will slow down the scanning process.